src/cephadm/cephadm

```diff
@@ -2145,6 +2145,29 @@ def command_unit():
 
 ##################################
 
+@infer_fsid
+def command_logs():
+    # type: () -> None
+    if not args.fsid:
+        raise Error('must pass --fsid to specify cluster')
+
+    (daemon_type, daemon_id) = args.name.split('.', 1)
+    unit_name = get_unit_name(args.fsid, daemon_type, daemon_id)
+
+    cmd = []
+    cmd.extend([find_program('journalctl')])
+    cmd.append('--no-pager')
```

Comment:
do we want to force --no-pager?  i've been finding it pretty useful.

Reply:
hrm, yeah a pager is useful, but it's also easy to pipe the output to less.

My main concern was a script that would hang, because it tried to simply invoke the command and read stdout ... but the script could pass the --no-pager option too.

Reply:
I wouldn't force it as people might want to use less or something else. If it's an option they can either use it or not but if we force it they have to.

Reply:
updated with `--no-pager` removed